app/src/test/kotlin/com/hedvig/app/feature/insurance/ui/detail/YourInfoItemsTest.kt

```diff
@@ -20,4 +20,16 @@ class YourInfoItemsTest {
     val viewState = INSURANCE_DATA_UPCOMING_AGREEMENT.contracts[0].toMemberDetailsViewState()
     assertThat(viewState.pendingAddressChange).isNotNull()
   }
+
+  @Test
+  fun `when termination flow is enabled, it should be part of the items`() {
+    val viewState = INSURANCE_DATA_UPCOMING_AGREEMENT.contracts[0].toMemberDetailsViewState(true)
+    assertThat(viewState.cancelInsurance).isNotNull()
+  }
+
+  @Test
+  fun `when termination flow isn't disabled, it should not be part of the items`() {
```

Comment:
Should this be "when termination flow is disabled ... "?

Reply:
Oops, addressed here https://github.com/HedvigInsurance/android/commit/237ade52be278fd50ed5281ead78f19dd5e5db5a